location/views.py

```diff
@@ -162,11 +165,17 @@ def favorites(request, pk):
     user = request.user
     if apartment not in user.favorites.all():
         user.favorites.add(apartment)
-        messages.success(request, "This apartment has been added to your favorites!")
+        messages.success(
+            request,
+            "This apartment has been added to your favorites!",
+            extra_tags="apartment",
+        )
```

Comment:
Where do these messages display? Not sure if I'm doing something wrong.

Reply:
good question. Actually these messages are not displayed (see my comment on `location.html`). But you should be able to see the message displayed when you contact the landlord.